daemon/inspect.go

```diff
@@ -151,3 +166,30 @@ func (daemon *Daemon) VolumeInspect(name string) (*types.Volume, error) {
 	}
 	return volumeToAPIType(v), nil
 }
+
+func getBackwardsCompatibleNetworkSettings(settings *network.Settings) *v1p20.NetworkSettings {
```

Comment:
@mrjana it shouldn't because `docker inspect` will talk API v1.21. This would only be the case if they did it through the API directly with a version < 1.21. But with that API they can't create new networks anyways.